pp/integrations/plugin-packs/procedures/network-firewalls-juniper-ssg-snmp.md

```diff
@@ -2,36 +2,627 @@
+| SNMP Agents     | Discover your resource through a SNMP subnet scan. You need to install the [Generic SNMP](./applications-protocol-snmp.md) connector to get the discovery rule and create a template mapper for the **Net-Juniper-SSG-custom** host template |
+
+More information about discovering hosts automatically is available on the [dedicated page](/docs/monitoring/discovery/hosts-discovery).
+
+#### Service discovery
+
+| Rule name                | Description |
+|:-------------------------|:------------|
+| Net-Juniper-SSG-Vpn-Name |             |
+
+More information about discovering services automatically is available on the [dedicated page](/docs/monitoring/discovery/services-discovery)
+and in the [following chapter](/docs/monitoring/discovery/services-discovery/#discovery-rules).
+
+### Collected metrics & status
+
+Here is the list of services for this connector, detailing all metrics linked to each service.
+
+<Tabs groupId="sync">
+<TabItem value="Cpu" label="Cpu">
+
+| Metric name | Unit  |
+|:------------|:------|
+| cpu_1min    | %     |
+| cpu_5min    | %     |
+| cpu_15min   | %     |
+
+</TabItem>
+<TabItem value="Hardware" label="Hardware">
+
+Coming soon
+
+</TabItem>
+<TabItem value="Memory" label="Memory">
+
+| Metric name | Unit  |
+|:------------|:------|
+| used        | B     |
+| fragmented  | N/A   |
+
+</TabItem>
+<TabItem value="Sessions" label="Sessions">
+
+| Metric name | Unit  |
+|:------------|:------|
+| usage       | N/A   |
+| failed      | /s    |
+
+> To obtain this new metric format, include **--use-new-perfdata** in the **EXTRAOPTIONS** service macro.
+
+</TabItem>
+<TabItem value="Traffic-*" label="Traffic-*">
+
+| Metric name                                          | Unit  |
+|:-----------------------------------------------------|:------|
+| *interface_name*#status                              | N/A   |
+| *interface_name*#interface.traffic.in.bitspersecond  | b/s   |
+| *interface_name*#interface.traffic.out.bitspersecond | b/s   |
+
+> Applies to the following service templates: Traffic-Generic-Id, Traffic-Generic-Name, Traffic-Global
+
+</TabItem>
+<TabItem value="Vpn-Status" label="Vpn-Status">
+
+| Metric name       | Unit  |
+|:------------------|:------|
+| *vpn*#status      | N/A   |
+| *vpn*#update-time | s     |
+
+> To obtain this new metric format, include **--use-new-perfdata** in the **EXTRAOPTIONS** service macro.
+
+</TabItem>
+<TabItem value="Vpn-Usage" label="Vpn-Usage">
+
+| Metric name       | Unit  |
+|:------------------|:------|
+| *vpn*#traffic-in  | b/s   |
+| *vpn*#traffic-out | b/s   |
+
+> To obtain this new metric format, include **--use-new-perfdata** in the **EXTRAOPTIONS** service macro.
+
+</TabItem>
+</Tabs>
 
 ## Prerequisites
 
-This chapter describes the prerequisites installation needed by plugins to run.
+### SNMP Configuration
+
+To use this pack, the SNMP service must be properly configured on your ressource.
```

Comment:
```suggestion
To use this pack, the SNMP service must be properly configured on your resource.
```

Reply:
What about:
The SNMP service must be configured and activated on the host. Please refer to the official documentation from the constructor/editor.